KinSDK/KinSDK/KinAccount.swift

```diff
@@ -63,7 +63,13 @@ public class KinAccount {
             let price = try? store.client.suggestGasPrice(store.context),
             let toAddress = GethNewInterface(),
             let value = GethNewInterface() else {
-                throw KinError.internalInconsistancy
+                throw KinError.internalInconsistency
+        }
+
+        let currentBalance = (try balance() as NSDecimalNumber).uint64Value
```

Comment:
what happen if try fails?

Reply:
the whole method `throws`, so if `balance()` fails, sending a transaction will fail as well.

Reply:
👍